astropy_helpers/sphinx/conf.py

```diff
@@ -135,15 +135,11 @@ def check_sphinx_version(expected_version):
     'sphinx.ext.coverage',
     'sphinx.ext.inheritance_diagram',
     'sphinx.ext.viewcode',
-    'astropy_helpers.sphinx.ext.numpydoc',
-    'astropy_helpers.sphinx.ext.astropyautosummary',
-    'astropy_helpers.sphinx.ext.autodoc_enhancements',
-    'astropy_helpers.sphinx.ext.automodsumm',
-    'astropy_helpers.sphinx.ext.automodapi',
+    'astropy_helpers.extern.numpydoc',
+    'astropy_helpers.extern.automodapi.automodapi',
     'astropy_helpers.sphinx.ext.tocdepthfix',
     'astropy_helpers.sphinx.ext.doctest',
-    'astropy_helpers.sphinx.ext.changelog_links',
-    'astropy_helpers.sphinx.ext.smart_resolver']
```

Comment:
Do we not need this any more? It has been moved into automodapi, but we don't do a blank inclusions of everything that is in there, do we?